tests/object_file_system_test.php

```diff
@@ -433,7 +432,6 @@ public function test_object_storage_deleter_can_delete_object_if_enabledelete_is
         $this->delete_draft_files($filehash);
         $this->filesystem->remove_file($filehash);
         $this->assertFalse($this->is_locally_readable_by_hash($filehash));
-        $this->assertFalse($this->is_externally_readable_by_hash($filehash));
     }
 
     public function test_object_storage_deleter_can_delete_object_if_enabledelete_is_off_and_object_is_remote() {
```

Comment:
I don't see any tests to cover the new functionality

Reply:
Added in further commit.